ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/VulnReports/HelpIconTh.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Flex, FlexItem, Tooltip } from '@patternfly/react-core';
+import { Th, ThProps } from '@patternfly/react-table';
+import { OutlinedQuestionCircleIcon } from '@patternfly/react-icons';
+
+type TooltipThProps = {
```

Comment:
Naming nit + export suggestion nit:
```suggestion
export type HelpIconThProps = {
```

Reply:
Oops, thanks for catching that